spec/examples/complex.schema.md

```diff
@@ -150,7 +150,7 @@ This is not foo.
 
 `nonfoo`
 * is optional
-* type: `boolean`
+* type: `const`
```

Comment:
@AndyOGo Why the change in type here?

Reply:
@anjankaur 
because it's schema defines it's boolean value by `const` here:
https://github.com/adobe/jsonschema2md/blob/4bf35ddacdd53a1e5d1112fb0f2a0f28e3568323/examples/schemas/abstract.schema.json#L21-L25

And the `simpletype` function checks for `const` prop with higher priority than the `type` prop, here:
https://github.com/adobe/jsonschema2md/blob/2cc177c681e4b63d0d2784b8bc2f373126ebb162/lib/markdownWriter.js#L94-L102